scenes/cslib_object.txt

```diff
@@ -36,12 +39,18 @@
 *comment *create dragon_1_hp 		100
 *comment *create dragon_2_name 		"Dave"
 *comment *create dragon_2_hp		150
+*comment *create dragon_max			2
```

Comment:
Check the indentation here: spaces vs tabs?

Reply:
OK